readme.md

```diff
@@ -8,127 +8,95 @@ binaries that have been lifted to intermediate formats. Supported input formats
+[INFO]   [!] Loading Program
+[INFO]   [!] Removing external function calls
+[INFO]   [!] Stripping unreachable
+[INFO]   [!] Removed 11 functions (1 remaining)
+[INFO]   [!] Translating to Boogie
+[INFO]   [!] Writing file...
 $ tail boogie_out.bpl 
     mem, Gamma_mem := memory_store32_le(mem, bvadd64(R9, 52bv64), R8[32:0]), gamma_store32(Gamma_mem, bvadd64(R9, 52bv64), Gamma_R8);
     assert ((bvadd64(R9, 52bv64) == $z_addr) ==> (L(mem, $x_addr) ==> Gamma_x_old));
     assert bvsge32(memory_load32_le(mem, $z_addr), z_old);
-    main_argv_out, Gamma_main_argv_out := R1, Gamma_R1;
-    main_result, Gamma_main_result := R0[32:0], Gamma_R0;
-    FP_out, Gamma_FP_out := R29, Gamma_R29;
-    LR_out, Gamma_LR_out := R30, Gamma_R30;
-    SP_out, Gamma_SP_out := R31, Gamma_R31;
+    assume {:captureState "%0000033f"} true;
+    goto main_basil_return;
+  main_basil_return:
+    assume {:captureState "main_basil_return"} true;
     return;
 }
+
 $ boogie boogie_out.bpl 
 
 Boogie program verifier finished with 4 verified, 0 errors
 ```
 
 ## Installation and use
 
-### Using Containers 
-
-(This method has only been tested on linux)
-
-The docker config can be used to provide bap with the asli-plugin as well as compile the tool itself.
-
-Requirements:
-
-- podman, podman-compose
-
-#### 1. Obtain the image image
-
-##### From github 
-
-`podman pull ghcr.io/uq-pac/basil-dev:latest`
-
-##### Build the images
+This is a Scala 3 project using the [mill](mill-build.com) build system, it can be run with `mill run`. 
 
-To build the images, from the root of the respository run
-
-```
-podman-compose build
-```
-
-#### 2. Use the images with compose 
-
-Individual services can be built with `podman compose build $servicename` from the root of the repo. 
-
-The services provided are:
+### Local Development 
 
-- `basil-dev` dev environment containng scala build environment, bap and cross-compilers
-   - To compile basil into the current directory using the sbt and scala provided by the docker image: 
-      - `podman compose run basil-dev sbt assembly`
-   - To enter a shell inside the container
-      - `podman compose run basil-build`
-- `basil-build`
-   - The same as above however containg only the scala build environment and compiled basil
-   - To recompile with the currently checked-out repo run `podman-compose build basil-build`
-   - To enter a shell inside the container
-      - `podman compose run basil-build`
-- `basil` precompiled jar file and tools
-   - To run the jar inside the docker image `podman-compose run basil-dev $arguments...`
-- `compiler-explorer`
-    - instance of [godbolt.org](godbolt.org) with the tools installed. 
 
-Or enter the dev container manually, mounting the current directory (the same as podman-compose basil-dev).
+The tool is a Scala project, and is therefore OS-independent, but is only tested under linux. 
```

Comment:
Not true, I am using Windows for development, with wsl used for all the compiling and lifting of binaries.

Reply:
@ailrst